usr/bin/cylc

```diff
@@ -16,54 +16,130 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-# Wrapper for selection between installed versions of cylc.
-# Author: Matt Shin (Met Office).
-
-#_____________________________
-# INSTALLATION: 
+#------------------------------------------------------------------------------
+# Wrapper script to support multiple Cylc versions installed on the same host.
+# Handles Conda and Python virtual envs, and plain Cylc 7 installations.
 #
-#  1. Install this script as "cylc" in $PATH for normal users.
+# WRAPPER INSTALLATION AND CONFIGURATION
+#---------------------------------------
+# Copy this script as "cylc" into the default $PATH on scheduler and job hosts,
+# and modify CYLC_HOME_ROOT(_ALT) below (see "EDIT ME") to point to the parent
+# directory of all installed versions.
 #
-#  2. Install cylc versions as described in the INSTALL file, e.g.:
-#     /opt/cylc-7.6.1/
-#     /opt/cylc-7.7.0/
-#     /opt/cylc -> cylc-7.7.0/  # symlink DEFAULT version
+# HOW IT WORKS
+#-------------
+# Intercept "cylc" commands and re-invoke them with the appropriate cylc-flow
+# version, according to location or version information in the environment:
+# + 1) $CYLC_HOME if defined, or
+# + 2) $CYLC_HOME_ROOT[_ALT]/$CYLC_VERSION
```

Comment:
Maybe have a third option

```
2) $CYLC_HOME_ROOT/$CYLC_VERSION
3) $CYLC_HOME_ROOT_ALT/$CYLC_VERSION
```
and/or describe that the `_ALT` option is used after `CYLC_HOME_ROOT`?

Reply:
OK, I guess my `ROOT[_ALT]` is a bit cryptic.

Reply:
Text tweaked.